src/main.c

```diff
@@ -1119,7 +1120,7 @@ main(int argc, char *argv[])
 {
     int ret = EXIT_SUCCESS;
     int c, *idx, i;
-    int daemonize = 1, verb = 0;
+    int daemonize = 1, verb = 0, use_syslog = 0;
```

Comment:
If you are adding a flag, use it correctly. By default (with no options), syslog is used so I would set it to `1`. Then, for the `d` option, it should be set to `0`.

Reply:
Fixed in updated PR (force-push)